libs/boost/Makefile

```diff
@@ -363,7 +386,7 @@ define Build/Compile
 			$(if $(CONFIG_boost-runtime-shared),runtime-link=shared,) \
 			$(if $(CONFIG_boost-runtime-static),runtime-link=static,) \
 			$(if $(CONFIG_boost-runtime-static-and-shared),runtime-link=shared$(comma)static,) \
-			$(if $(CONFIG_boost-single-thread),threading=single,) \
+			$(if $(CONFIG_boost-single-thread),threadingm=single,) \
```

Comment:
It's a typo. Thanks for noticing. It would only affect if someone was to compile Boost without multi-threading support.